mockery/generator.go

```diff
@@ -631,6 +631,8 @@ func (g *Generator) Write(w io.Writer) error {
 
 	res, err := imports.Process("mock.go", theBytes, opt)
 	if err != nil {
+		line := "--------------------------------------------------------------------------------------------"
+		fmt.Fprintf(os.Stderr, "Between the lines is the file (mock.go) mockery generated in-memory but detected as invalid:\n%s\n%s\n%s\n", line, g.buf.String(), line)
```

Comment:
I feel like this comment might be a little self evident and maybe even redundant assuming that the error message gets printed elsewhere.

It should be easy enough to figure out that the when I see an error message, and right above it is a whole go file printed to my console that that is the go file that is causing issues.

Just my opinion,